src/components/quote/Quote.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { H4 } from '../typography'
+import { QuoteProps } from './Quote.types'
+import { StyledBlockquote } from './Quote.styles'
+
+export const Quote: React.FC<QuoteProps> = ({ children }) => {
```

Comment:
We need to ensure the rest of the props are passed into the component for extension - currently we are just taking the `children` prop and no others.